Mods/vcmi/config/vcmi/german.json

```diff
@@ -361,4 +365,4 @@
 	"core.bonus.WATER_IMMUNITY.description": "Immun gegen alle Zauber der Wasserschule",
 	"core.bonus.WIDE_BREATH.name": "Breiter Atem",
 	"core.bonus.WIDE_BREATH.description": "Breiter Atem-Angriff (mehrere Felder)"
-}
+}
```

Comment:
Text files should end with a newline: https://stackoverflow.com/a/729795/2278742